apps/like/src/object/object.controller.ts

```diff
@@ -1,4 +1,15 @@
-import { Body, Controller, Get, Post, Put, ValidationPipe, HttpStatus, Param, Req, Res } from '@nestjs/common';
+import {
+  Body,
+  Controller,
+  Get,
+  Post,
+  Put,
+  ValidationPipe,
+  HttpStatus,
+  Param,
+  Req,
+  Res,
+} from '@nestjs/common';
```

Comment:
Maybe we should add this: https://github.com/Greenstand/treetracker-like/issues/34